web/src/components/SharingPopup.tsx

```diff
@@ -82,6 +79,7 @@ const TooltipContainer = styled.div<{
             transform: rotate(-180deg);
           `
         : css`
+            right: 11px;
```

Comment:
Shouldn't right be same as left (`20px`)?

Reply:
same comment as below